docs/news.d/cmake-build.rst

```diff
@@ -1,4 +1,4 @@
-.. news-prs: 4203
+.. news-prs: 4203 4304
```

Comment:
```suggestion
.. news-prs: 4203
```

4304 is related to CMake, but it's a minor documentation change that I don't consider to be news-worthy.